comms/rabbitmq.go

```diff
@@ -371,11 +382,17 @@ func (rmq rabbitMQComms) ListenForTasks(uuid string) {
 		rmq.config.AMQP.Port,
 	)
 
+	retries := 0
 	conn, err := amqp.Dial(queue_url)
-	if err != nil {
-		log.Error(err)
-		log.Error("Failed to connect to RabbitMQ")
-		os.Exit(1)
+	for err != nil {
+		if retries > connectRetry {
+			log.Error("Failed to connect to RabbitMQ")
+			return err
+		}
+
+		retries++
+		log.Warnf("Failure connecting to RabbitMQ - retry #%d", retries)
+		time.Sleep(1 * time.Second)
```

Comment:
Same issue here. Since the logic is identical it could be a candidate for it's own function.